packages/type-orm/src/decorators/order-options.decorator.ts

```diff
@@ -13,9 +13,11 @@ export function OrderOptions<T = any>(prefix?: string) {
     )(target, propertyKey, parameterIndex)}
 }
 
+export type TypeORMOrderByConditionValue = ("ASC"|"DESC")|{
+  order: "ASC"|"DESC";
+  nulls?: "NULLS FIRST"|"NULLS LAST";
```

Comment:
```suggestion
  nulls: "NULLS FIRST"|"NULLS LAST";
```

Reply:
(since `{order: "asc", nulls: undefined}` is precisely what we're trying to avoid)